themes/harper/layouts/shortcodes/figure.html

```
@@ -0,0 +1,107 @@
+{{/*
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)